app/src/main/java/ch/epfl/skysync/models/reports/CrewReport.kt

```diff
@@ -0,0 +1,22 @@
+package ch.epfl.skysync.models.reports
+
+import ch.epfl.skysync.models.flight.Vehicle
+import ch.epfl.skysync.models.user.Crew
+import java.sql.Time
+
+data class CrewReport(
+    val id: String,
+    val author: Crew,
+    val littleChampagne: Int,
+    val bigChampagne: Int,
+    val prestigeChampagne: Int,
+    val littleChampagneToFarmer: Boolean,
+    val bigChampagneToFarmer: Boolean,
+    val prestigeChampagneToFarmer: Boolean,
```

Comment:
Isn't having three different fields to specify the type of champagne given to the farmer way too specific ?
Wouldn't this be a good use case of the `comments` fields ?

Reply:
Yeah, I've removed it.